.ci/ci_functions.sh

```diff
@@ -0,0 +1,46 @@
+function configure () {
+    rm -rf $NFS_WORKSPACE
+    rm -rf ./nccl-rdma-sharp-plugins
+    cd $NFS_WORKSPACE_ROOT
+    git clone git@github.com:Mellanox/nccl-rdma-sharp-plugins.git 
```

Comment:
If it's clone just for the cfg files, maybe make it shallow?

Reply:
Code was changed